lib/shell/make_externals.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash   
+set -e
+
+MAKE_EXTERNALS_SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+REPO_ROOT=$(realpath "$MAKE_EXTERNALS_SCRIPT_DIR/../../")
+
+source "${REPO_ROOT}/local_env.sh"
+
+# Check if there are valid paths in local_env
+create_links=false
+for target_path in "${EXTERNAL_PATHS[@]}"; do
```

Comment:
I think the error message "Target path $target_path does not exist for $name" will only populated up if there are both at least one valid path for external target and both at least one wrong path.

What if all `target_path`(s) in `local_env.sh` don't exist? I think then `create_links` will return `false`, but program will exit without any error message. I suggest also add error/warning message handling for this case.

Reply:
Thanks @ShiqiYang2022 , that's a good point. I have pushed a change to the if-statement logic in 8913f7e which I think will solve this.

Reply:
@lucamlouzada That looks perfect, thanks!